### DISCUSSION TITLE
Lost my Github Account

### DISCUSSION THREAD

**Discussion by shashanksjSSJ** [community/community#86672] [Other Feature Feedback, Questions, & Ideas]
Hello, 
My question😁, a few months ago, I had used my college email id to signup to GitHub, after logging in, I did not perform the email verification, and when I realized that I could use my college email id to access GitHub student pack, I tried to login with my college Gmail id, but then I had forgotten my password. I cannot reset my password because I have not verified my account, due to which my reset password Gmail is not being sent to me. 
What Do I do?? I want to access my GitHub account associated with my College Gmail ID.

**Comment by ayaanvasudeva** [comment]
Go over here to find useful information https://github.com/feathersjs-ecosystem/feathers-authentication-management/issues/41
Hope this helps

**Comment by kmadaminjanova0508** [comment]
Hi, open the GitHub Support form at https://support.github.com/contact. Select the "I can't sign in" category, and then choose "I'm having problems with my email" and "I need to update my email address" as the subcategories. I am sure u will get response. :)
